pytx3/pytx3/signal_type/signal_base.py

```diff
@@ -103,6 +111,7 @@ class SimpleSignalType(SignalType, HashMatcher):
 
     def __init__(self) -> None:
         self.state: t.Dict[str, SimpleDescriptorRollup] = {}
+        self.indicator_state: t.Dict[int, ThreatIndicator] = {}
```

Comment:
Why does indicator state have int key but descriptor state has string key?

Reply:
indicator_state is keyed by indicator id, state (for descriptors) is keyed by the indicator (only 1 descriptor is stored per indicator)